app/backend/src/couchers/migrations/versions/8b6297128973_add_foreign_key_indexes.py

```diff
@@ -0,0 +1,66 @@
+"""Modify the database
```

Comment:
When you run the command in the docs, you can give it a message, and it'll name the file accordingly.

Reply:
I know that, but found it easier to rename later, so I could run the script from a button press in my ide. But now I see that  I would need to chage it in two places, your advise is the way to go.